### ISSUE TITLE
Pods PodsDFV error when extending media

### ISSUE THREAD

**Issue by codente** [pods-framework/pods#5799]
**Describe the bug**
Similar or same issue as #5313.  We are seeing an issue with Beaver Builder when taking the steps below.   (Note we also saw the same when testing with Elementor)

**To Reproduce**

1. Install BB Lite
2. Pods Admin > Add New > Extend Existing
3. Set Content Type to Media
4. Just add any field and save the Pod
5. Drop a Photo module to a page then try selecting any of the images. You can click on an image but "Select Photo" is not clickable and you see the error about PodsDFV being undefined in console.

**Expected behavior**
That "Select Photo" button is clickable

**Screenshots**
<img width="1516" alt="Image 2020-08-07 at 12 54 32 PM" src="https://user-images.githubusercontent.com/2265866/89674095-2b8faf00-d8ad-11ea-8336-79bfd5edec39.png">

## Pods Version
2.7.21

## WordPress Environment
<details>
WordPress Version: 5.4.2

PHP Version: 7.4.6

MySQL Version: 5.5.5

Server Software: nginx/1.19.0

Your User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36

Session Save Path:

Session Save Path Exists: No

Session Save Path Writeable: No

Session Max Lifetime: 1440

Opcode Cache:

Apc: No
Memcached: No
OPcache: Yes
Redis: No
Object Cache:

APC: No
APCu: No
Memcache: Yes
Memcached: Yes
Redis: Yes
WPDB Prefix: test_

WP Multisite Mode: No

WP Memory Limit: 40M

Current Memory Usage: 22.260M

Current Memory Usage (real): 2.000M

Pods Network-Wide Activated: No

Pods Install Location: /home/test/webapps/test/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: Beaver Builder Theme

Currently Active Plugins:

Beaver Builder Plugin (Lite Version): 2.3.2.5
Pods - Custom Content Types and Fields: 2.7.21

</details>

## Pods Package Export (helpful!)

```
{"meta":{"version":"2.7.21","build":1596822965},"pods":{"2776":{"id":2776,"name":"media","label":"Media","description":"","type":"media","storage":"meta","object":"media","alias":"","fields":{"text":{"id":2777,"name":"text","label":"Text","description":"","help":"","class":"","type":"text","weight":0,"pick_object":"","pick_val":"","sister_id":"","required":"0","text_allow_shortcode":"0","text_allow_html":"0","text_allowed_html_tags":"strong em a ul ol li b i","text_max_length":"255","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","text_repeatable":"0","text_placeholder":""}},"show_in_menu":1,"rest_enable":"0","read_all":"0","write_all":"0"}}}
```

**Comment by JoryHogeveen** [comment]
It seems Elementor removes all scripts and styles from other plugins. This is the reason it's not working.
I've been busy making this compatible but once I re-add our scripts through Elementor there are some JS conflicts happening that prevents Elementor from working at all.
Might be a difficult one to fix...

**Comment by JoryHogeveen** [comment]
Fixed for Beaver Builder in 2.7.23. To be continued for other page builders in #5864

**Comment by codente** [comment]
Thanks @JoryHogeveen!
